src/routes/RequestsRoute.js

```diff
@@ -800,12 +800,16 @@ class RequestsRoute extends React.Component {
   }
 
   renderFilters = (onChange) => {
+    const { resources } = this.props;
+    const { titleLevelRequestsFeatureEnabled = false } = getTlrSettings(resources.configs.records[0]?.value);
+
     return (
       <RequestsFilters
         activeFilters={this.getActiveFilters()}
         resources={this.props.resources}
```

Comment:
Can we here pass `resources` instead of `this.props.resources` since we are destructuring above?

Reply:
Done